platform/view/services/storage/driver/sql/postgres/sql_test.go

```diff
@@ -49,3 +55,86 @@ func TestPostgres(t *testing.T) {
 		return p.(*KeyValueStore).KeyValueStore
 	})
 }
+
+func verifyExtendedLoggingInCommand(t *testing.T, config *ContainerConfig, expectExtendedLogging bool) {
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	require.NoError(t, err)
+
+	ctx := context.Background()
```

Comment:
you can use `t.Context()` as this allows the test harness to cancel the context and all probably all consumers of the context.